src/code42cli/cmds/search/extraction.py

```diff
@@ -96,3 +107,9 @@ def create_time_range_filter(filter_cls, begin_date=None, end_date=None):
 
     elif end_date and not begin_date:
         return filter_cls.on_or_before(end_date)
+
+
+def _process_events(output_format, include_all, events, output_header):
+    if include_all:
```

Comment:
Note that this means every 10,000 records, we are going to print the headers again...

This is not a big deal because we are also going to reach the END line of terminal pagers.... which is weird, but probably the best way of doing this currently.